tests/benchmarks/garage/tf/baselines/benchmark_gaussian_cnn_baseline.py

```diff
@@ -1,147 +0,0 @@
-from garage.experiment import deterministic
-from garage.tf.algos import PPO
-from garage.tf.baselines import GaussianCNNBaseline
-from garage.tf.envs import TfEnv
-from garage.tf.experiment import LocalTFRunner
-from garage.tf.policies import CategoricalCNNPolicy
-from tests.fixtures import snapshot_config
-import tests.helpers as Rh
-
-num_of_trials = 3
-
-params = {
-    'conv_filters': (32, 64, 64),
-    'conv_filter_sizes': (5, 3, 2),
-    'conv_strides': (4, 2, 1),
-    'conv_pad': 'VALID',
-    'hidden_sizes': (256, ),
-    'n_epochs': 1000,
-    'batch_size': 2048,
-    'use_trust_region': True
-}
-
-
-class BenchmarkGaussianCNNBaseline:
-    '''Benchmark GaussianCNNBaseline.'''
-
-    def benchmark_gaussian_cnn_baseline(self):
-        timestamp = datetime.datetime.now().strftime('%Y-%m-%d-%H-%M-%S-%f')
-        benchmark_dir = './data/local/benchmarks/ppo/%s/' % timestamp
-        for env_id in ['CubeCrash-v0', 'MemorizeDigits-v0']:
-            env = gym.make(env_id)
-            seeds = random.sample(range(100), num_of_trials)
-
-            task_dir = osp.join(benchmark_dir, env_id)
-            plt_file = osp.join(
-                benchmark_dir,
-                '{}_benchmark_gaussian_cnn_baseline.png'.format(env_id))
-            garage_csvs = []
-
-            for trial in range(num_of_trials):
-                seed = seeds[trial]
-
-                trial_dir = task_dir + '/trial_%d_seed_%d' % (trial + 1, seed)
-                garage_dir = trial_dir + '/garage'
-
-                with tf.Graph().as_default():
-                    # Run garage algorithms
-                    env.reset()
-                    garage_csv = run_garage(env, seed, garage_dir)
-                garage_csvs.append(garage_csv)
-
-            env.close()
-
-            Rh.relplot(g_csvs=garage_csvs,
-                       b_csvs=[],
-                       g_x='Iteration',
-                       g_y='Evaluation/AverageReturn',
-                       g_z='Garage',
-                       b_x='Iteration',
-                       b_y='Evaluation/AverageReturn',
-                       b_z='GarageWithModel',
-                       trials=num_of_trials,
-                       seeds=seeds,
-                       plt_file=plt_file,
-                       env_id=env_id,
-                       x_label='Iteration',
-                       y_label='Evaluation/AverageReturn')
-
-
-def run_garage(env, seed, log_dir):
-    '''
-    Create garage model and training.
-
-    Replace the ppo with the algorithm you want to run.
-
-    :param env: Environment of the task.
-    :param seed: Random seed for the trial.
-    :param log_dir: Log dir path.
-    :return:
-    '''
-    deterministic.set_seed(seed)
-    config = tf.compat.v1.ConfigProto(allow_soft_placement=True,
-                                      intra_op_parallelism_threads=12,
-                                      inter_op_parallelism_threads=12)
-    sess = tf.compat.v1.Session(config=config)
```

Comment:
@ahtsan Is it safe to use the default session?

Reply:
I think so within `LocalTFRunner` scope